### ISSUE TITLE
Use better volume examples

### ISSUE THREAD

**Issue by owainlewis** [oracle/mysql-operator#93]
Currently the examples used for volumes use a hostpath volume. This has, from experience, led to some confusion since it generally won't work without modification of the example. We would be better off using a different example for volumes.

**Comment by bweston92** [comment]
Can we have an example for GKE too, just upgraded a node pool and lost everything as I didn't realise it was using host path aha

**Comment by owainlewis** [comment]
@bweston92 yes we'll update the examples to work on any K8s cluster. That would make it easier for users looking to get started quickly.
